AndroidNativeLoginTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,8 @@
 	android:versionName="1.0"
 	android:installLocation="internalOnly">
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
This is required for Google's reCAPTCHA.

Reply:
This is required if you use sockets... TIL.